src/App.tsx

```diff
@@ -14,11 +14,34 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  showGoods: boolean;
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    showGoods: false,
```

Comment:
sounds like a function
```suggestion
    shouldShowGoods: false,
```